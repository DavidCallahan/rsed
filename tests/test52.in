mv -f '_build/opt/common/fbid/be3b022/libdbid.a'.tmp '_build/opt/common/fbid/be3b022/libdbid.a'
END_TIME=`date +%s.%N`
_msg ""
 0.01s _build/opt/common/fbid/be3b022/libdbid.a
 9.23s admarket/adsearch2/query/QueryEngineBenchmark.cpp
In file included from admarket/adsearch2/store/ShardStore.cpp:3:
./admarket/adsearch2/store/ShardStore.h:54:16: error: private field 'family_' is not used [-Werror,-Wunused-private-field]
    DataFamily family_;
               ^
./admarket/adsearch2/store/ShardStore.h:88:16: error: private field 'family_' is not used [-Werror,-Wunused-private-field]
    DataFamily family_;
               ^
2 errors generated.
11.86s error: admarket/adsearch2/store/ShardStore.cpp
fbmake: *** [_build/opt/admarket/adsearch2/store/ee1f113/ShardStore.o] Error 1
Fbmake run id: D9YarmDzGOf
