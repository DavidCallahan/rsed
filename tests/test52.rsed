foreach all
  skip to "^In file included from"
  copy past "^[0-9]+ errors? generated"
end
