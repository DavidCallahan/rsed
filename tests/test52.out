In file included from admarket/adsearch2/store/ShardStore.cpp:3:
./admarket/adsearch2/store/ShardStore.h:54:16: error: private field 'family_' is not used [-Werror,-Wunused-private-field]
    DataFamily family_;
               ^
./admarket/adsearch2/store/ShardStore.h:88:16: error: private field 'family_' is not used [-Werror,-Wunused-private-field]
    DataFamily family_;
               ^
2 errors generated.
